src/entercommand.js

```diff
@@ -28,14 +35,9 @@ export default class EnterCommand extends Command {
  * Creates a new block in the way that the <kbd>Enter</kbd> key is expected to work.
  *
  * @param {engine.model.Batch} batch A batch to which the deltas will be added.
- * @param {engine.model.Selection} selection
- * @param {Object} options
- * @param {Boolean} options.defaultBlockName The name of the block which should be created when Enter leaves
- * another block. Usually set to `'paragraph'`. For example, when creating a block in `<heading>foo^</heading>`,
- * the result will be `<heading>foo</heading><paragraph>^</paragraph>`.
+ * @param {engine.model.Selection} selection Selection on which the action should be performed.
  */
-export function enterBlock( batch, selection, options = {} ) {
-	const defaultBlockName = options.defaultBlockName;
+function enterBlock( batch, selection ) {
```

Comment:
Why you cannot integrate undo steps? If the command adds the used `batch` to the `data` object passed with `afterExecute` event callback you can add new deltas to it.

Of course this way you cannot do `beforeExecute` and this could be an argument for making it more flexible.

Anyway, I don't think this need to be exposed anymore as calling this method is everything what `EnterCommand` does. There are no options, etc. So if you want to call this method, you can call `editor.execute( 'enter' )` too. If you want to modify this, use `afterExecute` callback. My rule of thumb was "if `Headings` does not need it, it does not need to be exposed".